app/services/streaming/streaming.ts

```diff
@@ -262,7 +262,7 @@ export class StreamingService extends StatefulService<IStreamingServiceState>
       queryParams: {},
       size: {
         width: 600,
-        height: 550,
+        height: 620,
```

Comment:
shouldn't the height of this depend on if the twitter component is visible?

Reply:
The twitter component is always visible if you're logged in. If you're not logged in the edit stream info window doesn't show.

Reply:
kk wasn't sure about that, thanks for clarifying